packs/menace-under-otari-bestiary/minotaur-hunter-bb.json

```
@@ -0,0 +1,472 @@
+                    "value": []
+                }
+            },
+            "type": "action"
+        },
+        {
+            "_id": "3STgLL5iZEI8pSTn",
+            "img": "systems/pf2e/icons/default-icons/action.svg",
+            "name": "Axe Swipe",
+            "sort": 500000,
+            "system": {
+                "actionType": {
+                    "value": "action"
+                },
+                "actions": {
+                    "value": 2
+                },
+                "category": null,
+                "description": {
+                    "value": "<p>The minotaur makes two Strikes with its greataxe against two adjacent enemies, both of whom are within its reach. The second attack uses the same multiple attack penalty as the first, but attacks it makes after this take a -10 multiple attack penalty.</p>"
+                },
+                "publication": {
+                    "license": "OGL",
+                    "remaster": false,
+                    "title": ""
+                },
+                "rules": [],
+                "slug": null,
+                "traits": {
+                    "value": []
+                }
+            },
+            "type": "action"
+        },
+        {
+            "_id": "fOwVEwaZPmB4enaL",
+            "img": "systems/pf2e/icons/default-icons/action.svg",
+            "name": "Perfect Orienteering",
+            "sort": 600000,
+            "system": {
+                "actionType": {
+                    "value": "passive"
+                },
+                "actions": {
+                    "value": null
+                },
+                "category": null,
+                "description": {
+                    "value": "<p>A minotaur automatically critically succeeds on Survival checks to @UUID[Compendium.pf2e.actionspf2e.Item.Sense Direction] or @UUID[Compendium.pf2e.actionspf2e.Item.Track]. </p>"
+                },
+                "publication": {
+                    "license": "OGL",
+                    "remaster": false,
+                    "title": ""
+                },
+                "rules": [],
+                "slug": null,
+                "traits": {
+                    "value": []
+                }
+            },
+            "type": "action"
+        },
+        {
+            "_id": "bQRnxgh1VHMUoffw",
+            "img": "systems/pf2e/icons/default-icons/action.svg",
+            "name": "Powerful Charge",
+            "sort": 700000,
+            "system": {
+                "actionType": {
+                    "value": "action"
+                },
+                "actions": {
+                    "value": 2
+                },
+                "category": null,
+                "description": {
+                    "value": "<p>The minotaur Strides twice, then makes a horn Strike. If the moved at least 20 feet from their starting position, the Strike's damage is increased to [[/r 2d8+10[piercing]]]</p>"
```

```suggestion
                    "value": "<p>The minotaur Strides twice, then makes a horn Strike. If the moved at least 20 feet from their starting position, the Strike's damage is increased to 2d8+10 piercing</p>"
```
We can actually automate this one.
```json
{
    "domain": "all",
    "key": "RollOption",
    "option": "powerful-charge",
    "toggleable": true
}
```
```json
{
    "key": "DamageDice",
    "predicate": ["powerful-charge"],
    "selector": "horn-damage",
    "diceNumber": 1    
}
```
```json
{
    "key": "FlatModifier",
    "selector": "horn-damage",
    "value": 2,
    "predicate": ["powerful-charge"]
}
```

Also make sure you're using the latest version of the data entry script to make those inline damage rolls `@Damage`